utils/download-references.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
```

Comment:
Maybe we should add parameter to specify the desired path as an argument in the script.